rest/replicatortest/replicator_test.go

```diff
@@ -4501,8 +4499,9 @@ func TestActiveReplicatorPushAndPullConflict(t *testing.T) {
 			// Start the replicator (implicit connect)
 			assert.NoError(t, ar.Start(ctx1))
 			// wait for the document originally written to rt2 to arrive at rt1.  Should end up as winner under default conflict resolution
-			base.WaitForStat(t, func() int64 {
-				return ar.GetStatus(ctx1).DocsWritten
+			base.RequireWaitForStat(t, func() int64 {
+				val := ar.GetStatus(ctx1).DocsWritten + ar.GetStatus(ctx1).DocWriteConflict
```

Comment:
Did you mean to change the return statement here to add `DocWriteConflict`?

Reply:
Yes, although I've improved it with the latest commit. It's a source of flakyness in this test where `DocsWritten` isn't enough to ensure the replication has completed.